.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: node_js
+
 node_js:
-  - "0.11"
-  - "0.10"
+    - 4.6
+    - 6.9
+
 notifications:
-  email: false
+    email: false
+
 before_install: npm install -g grunt-cli
-sudo: false
```

Comment:
`sudo: false` should be the default, so I removed it. Looking at it, it's the default for repos from 2015 or later. Not sure when this was created.